documentation/GETTING_STARTED.md

```diff
@@ -0,0 +1,349 @@
+
+### Viewing datasets via the web
+
+> **_NOTE:_** To use a dataset with our public build of Timelapse Feature Explorer, the dataset must be accessible using the HTTPS protocol (e.g., `https://example.com/your-dataset/`). If you need to use HTTP, run a local instance of the viewer.
+
+For this tutorial, you can load a pre-processed example copy of the dataset, which we've hosted on GitHub. You can access it at this URL: `https://raw.githubusercontent.com/allen-cell-animated/colorizer-data/doc/getting-started-guide/documentation/getting_started_guide/example/processed_dataset/manifest.json`
+
+If you have updated the dataset files or want to use a local dataset, skip to the next section ([Installing Timelapse Feature Explorer locally](#installing-timelapse-feature-explorer-locally)).
+
+#### Opening your dataset
+
+1. Open Timelapse Feature Explorer at [https://timelapse.allencell.org](https://timelapse.allencell.org).
+
+2. Click the **Load** in the header and paste in the following URL: `https://raw.githubusercontent.com/allen-cell-animated/colorizer-data/doc/getting-started-guide/documentation/getting_started_guide/example/processed_dataset/manifest.json`
+
+![The Load button on the Timelapse Feature Explorer header, next to the Help dropdown.](./getting_started_guide/assets/load-button.png)
+
+> **_NOTE:_** You can either provide the URL of the directory containing a `manifest.json` or the full URL path of a `.json` file that follows the [manifest specification](./DATA_FORMAT.md#dataset). We recommend specifying the full URL path that includes the `manifest.json`.
+
+Click **Load** in the popup menu to load the dataset. The viewer should appear with the dataset loaded!
+
+### Installing Timelapse Feature Explorer locally
+
+To view our **locally converted dataset**, we'll also need to run a **local version** of the Timelapse Feature Explorer.
+
+<details>
+<summary><b>[Why do we need to run a local instance of the viewer?]</b></summary>
+
+---
+The public version of TFE is served over HTTPS, which is a secure protocol. For security reasons, HTTPS pages cannot load HTTP content, which means that the public version of TFE can only access web content hosted on HTTPS.
+
+We can run a local server to serve our local files over HTTP, but we need to also run TFE on HTTP to access them.
+
+---
+
+</details>
+
+#### 1. TFE installation
+
+Install Node (v20 or above) from [nodejs.org](https://nodejs.org/). Once Node is installed, **open a command terminal** in the directory you want to install TFE into. Run the following commands to clone the repository and install the dependencies:
+
+```bash
+git clone https://github.com/allen-cell-animated/timelapse-colorizer.git
+cd timelapse-colorizer
+npm install
+npm run start
+```
+
+You can now run `npm run start` at anytime to start the viewer. By default, it will be mounted at `http://localhost:5173`.
```

Comment:
```suggestion
You can now run `npm run start` at anytime in this directory to start the viewer. By default, it will be mounted at `http://localhost:5173`.
```